docs/UserGuide.md

```diff
@@ -41,7 +58,7 @@ GithubContact is an address book **integrated with Github's Public API**, target
 **:information_source: Notes about the command format:**<br>
```

Comment:
Could you add a note to mention about prefix alias here? For example, running `add name/NAME` or `add n/NAME`  both works. Probably a table. You can refer to [here](https://github.com/AY2223S1-CS2103T-W08-2/tp/blob/master/src/main/java/seedu/address/logic/parser/CliSyntax.java)

Reply:
Added note on prefix aliases as well as a table listing all aliases for each attribute.